packages/eslint-plugin-orbit-components/docs/rules/custom-colors.md

```diff
@@ -0,0 +1,23 @@
+# custom-colors
+
+Prevents inconsistencies between Orbit and custom colors
+
+## Rule details
+
+The following patterns are considered warnings:
+
+```jsx
+const StyledWrapper = styled.div`
+  color: #808080;
+  background: #f1f1f1;
```

Comment:
With `#f1f1f1` it looks like it should be replaced with `theme.orbit.paletteWhite`, but that color is not white.
```suggestion
  background: #fff;
```

Reply:
I would just the documentation example here, but definitely not bother with similar colors. We might implement color token suggestions later 🙂